tests/neg-custom-args/no-experimental/experimentalType.scala

```diff
@@ -0,0 +1,31 @@
+import scala.annotation.experimental
+
+@experimental // error
```

Comment:
Looks like we don't have any test for an `@experimental case class`, I think we need to make sure that the generated companion is also marked `@experimental` (or else just always check both the class and the companion since having one experimental but not the other isn't very useful).

Reply:
Added the annotation in `annotateExperimental`.